terraform/api_gateway.tf

```diff
@@ -102,9 +102,21 @@ module "api_gateway" {
   integrations = {
     "POST /graphql" = {
       lambda_arn = module.lambda_function-graphql.lambda_function_arn
+      authorizer_key   = "passage"
     }
     "GET /graphql" = {
       lambda_arn = module.lambda_function-graphql.lambda_function_arn
+      authorizer_key   = "passage"
+    }
+  }
+
+  authorizers = {
```

Comment:
@TylerHendrickson want to get your eyes on this configuration. I read through the following documentation to set this up.

- [terraform docs](https://github.com/terraform-aws-modules/terraform-aws-apigateway-v2?tab=readme-ov-file#http-api-gateway)
- [passage docs](https://docs.passage.id/cloud-native/api-gateways/aws-api-gateway)